src/launcher.ts

```diff
@@ -87,7 +87,8 @@ export default class VSCodeServiceLauncher {
             }
 
             const version = cap[VSCODE_CAPABILITY_KEY].version || cap.browserVersion || DEFAULT_CHANNEL
-
+            cap[VSCODE_CAPABILITY_KEY].version = version
```

Comment:
`version` is not a valid capability. What is the idea behind this change?

Reply:
From line 89 I assumed that version was a valid capability, as it was used there. 

The idea was to store the actual vscode version before `cap.browserVersion` gets overwritten here: https://github.com/webdriverio-community/wdio-vscode-service/blob/main/src/launcher.ts#L174 
such that it is available here for selection of the right locators:
https://github.com/webdriverio-community/wdio-vscode-service/blob/main/src/service.ts#L207
(see https://github.com/webdriverio-community/wdio-vscode-service/pull/138/files#diff-84b68bce2abfcb25c3328df0286963d7dacc924940adc41923bf4f502a6bc95aR206)

Currently the `capabilities.browserVersion` is set to the chrome version, and the locators are thus not selected correctly.

Reply:
@jan872 actually this is correct, it is just changing the `VSCODE_CAPABILITY_KEY` capability object which can have arbitrary content.